app/_src/kic-v2/references/cli-arguments-2.12.x.md

```diff
@@ -74,9 +77,9 @@ and not CLI flags.
 | `--kong-admin-tls-client-key-file` | `string` | MTLS client key file for authentication. |  |
 | `--kong-admin-tls-server-name` | `string` | SNI name to use to verify the certificate presented by Kong in TLS. |  |
 | `--kong-admin-tls-skip-verify` | `bool` | Disable verification of TLS certificate of Kong's Admin endpoint. | `false` |
-| `--kong-admin-token` | `string` | The {{site.ee_product_name}} RBAC token used by the controller. |  |
+| `--kong-admin-token` | `string` | The Kong Enterprise RBAC token used by the controller. |  |
```

Comment:
```suggestion
| `--kong-admin-token` | `string` | The {{site.ee_product_name}} RBAC token used by the controller. |  |
```
This variable is correct in this instance (the name changed a little while ago to "Kong Gateway Enterprise", which is what this variable renders as)

Reply:
Thanks! Should I update the origin template in KIC's repo? (This file was generated from a template)